### PR TITLE
[Graphs] Y-Axis Labels displays M or K on respective values

### LINKED ISSUE #1560: [Graphs] Display Y-axis labels in the appropriate units
Labels for graphs with y-values in the millions are being displayed as ex. 1000K instead of 1M. 
 
Explore building a custom formatter function to remedy this. 
https://www.highcharts.com/forum/viewtopic.php?t=34647

### PR DESCRIPTION
**Pull request by Michaeldremy**
Closes #1560

### CODE REVIEW

File: src/data-browser/graphs/highchartsConfig.js [hunk 1]
```diff
@@ -151,25 +152,39 @@ export const deriveHighchartsConfig = ({
...
+      formatter: (tick) => {
+        if (!tick) return
+
+        if (tick.value > 999 && tick.value < 1000000) {
+          return (tick.value / 1000).toFixed(0) + "K" // convert to K for number from > 1000 < 1 million
+        } else if (tick.value >= 1000000) {
+          return (tick.value / 1000000).toFixed(1) + "M" // convert to M for number from > 1 million
+        } else if (tick.value < 900) {
+          return tick.value // if value < 1000, nothing to do
+        }
```
Comment (meissadia) [inline src/data-browser/graphs/highchartsConfig.js:167]: What happens if the value is `950`?
 
I don't think we need any comparison here.  Just, in all other cases, return the plain value. 
```suggestion
        } else {
          return tick.value // if value < 1000, nothing to do
        }
```

**Review by meissadia — CHANGES_REQUESTED** [review]
Looks pretty good.  One change.

File: src/data-browser/graphs/highchartsConfig.js (hunk 1 above)
Comment (Michaeldremy) [reply src/data-browser/graphs/highchartsConfig.js:167]: Code has been updated!

**Review by meissadia — APPROVED** [review]
Looks good!

**Pull request merged**